### PR TITLE
Add Services

### PR DESCRIPTION
**Pull request by BaVilius**
- Added an outline for the Services controller. Any upcoming extra logic will be moved to ServiceService.cs.
- Added a 'dummy' implementation for our tax system
- Smol fix in OrderController since it was causing build issues.

### CODE REVIEW

File: PspPos/Commons/TaxSystem.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using PspPos.Models;
+
+namespace PspPos.Commons
+{
+    public class TaxSystem
+    {
+        public readonly static float Tax = 0.1f;
```
Comment (naglissul) [inline PspPos/Commons/TaxSystem.cs]: Komentaro kažkokio plz. Ir also gal procentais rasyt geriau?

File: PspPos/Models/ServiceModel.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿namespace PspPos.Models
+{
+    public class Service
+    {
+        public string Name { get; set; }
+        public double Price { get; set; }
+        public string Description { get; set; }
```
Comment (naglissul) [inline PspPos/Models/ServiceModel.cs]: Maybe make these things nullable. Not so important, but would be cool

File: PspPos/Repositories/ServiceRepository.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.EntityFrameworkCore;
+using PspPos.Data;
+using PspPos.Models;
+using System.Linq.Expressions;
+
+namespace PspPos.Repositories
+{
+    public class ServiceRepository: IServiceRepository
```
Comment (naglissul) [inline PspPos/Repositories/ServiceRepository.cs:8]: Amm... so we're making repos?

File: PspPos/Commons/TaxSystem.cs (hunk 1 above)
Comment (BaVilius) [reply PspPos/Commons/TaxSystem.cs]: Kodel procentais geriau?

File: PspPos/Models/ServiceModel.cs (hunk 2 above)
Comment (BaVilius) [reply PspPos/Models/ServiceModel.cs]: Yep, pritariu

File: PspPos/Repositories/ServiceRepository.cs (hunk 3 above)
Comment (BaVilius) [reply PspPos/Repositories/ServiceRepository.cs:8]: Galvojau, kad kaip ir sutarem, kad visi paliksim savo implementacija kas kaip turim padare. Jei bus laiko galesiu pakeist po to

File: PspPos/Commons/TaxSystem.cs (hunk 1 above)
Comment (naglissul) [reply PspPos/Commons/TaxSystem.cs]: idk, gal skaitomiau. Bet ig jei komentara parasysi tai visaip ok. Also, procentais tu rasai prie discount, tai gal vienodai reiktu

Comment (BaVilius) [reply PspPos/Commons/TaxSystem.cs]: okay, turetu buti aiskiau. Aciu

**Review by naglissul — APPROVED** [review]
nice. sori kad uztrukau

**Pull request merged**